task3/PizzaTime/PizzaEntities/AbstractPizza.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PizzaTime.PizzaEntities
+{
+    public abstract class AbstractPizza
```

Comment:
No need for a class for each pizza type. All these different characteristics of pizza could be set as properties of a Pizza object, no other differences here.
Furthermore, if someone would like to add a new pizza type - they could do that in program instead of asking a developer to add a class for it manually.